website/content/_index.md

```diff
@@ -120,12 +123,89 @@ Config:
 
 We can interrogate the container more closely with `wexec`
 ```
-wash$ wexec 45a0265546d63a8f1b0d17033748db1468dc49dfd09cdaf2db62c45a60e82aaf whoami
+wash$ wexec swarm_web_1 whoami
 root
 ```
 
 Try exploring `docker/volumes` to interact with the volume created for Redis.
 
+### Finding with metadata
+
+Wash includes a powerful `find` command that can match based on an entry's metadata. For example, if we wanted to see what containers were created today, we would look at the `.Created` entry for Docker containers and the `.metadata.creationTimestamp` for Kubernetes pods. We can find all instances created in the last 24 hours with
+
+```
+find -meta .Created -24h -o -meta .metadata.creationTimestamp -24h
+```
+
+That says: list entries with the `Created` metadata - interpreted as a date-time - that falls within the last 24 hours, or that have the `metadata: creationTimestamp` in the last 24 hours. See `help find` for more on using `find`.
+
+If you want to try this out, or explore more Kubernetes functionality, you can create a Redis server with a persistent volume using Kubernetes in Docker and the following config:
+
+```
+cat <<EOF | kubectl create -f -
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: redis
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 100Mi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/mnt/data"
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: redis
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 50Mi
+---
+kind: Pod
+apiVersion: v1
+metadata:
+  name: redis
+spec:
+  containers:
+  - name: redis
+    image: redis
+    args: ["--appendonly", "yes"]
+    volumeMounts:
+    - name: redis
+      mountPath: /data
+  volumes:
+  - name: redis
+    persistentVolumeClaim:
+      claimName: redis
+EOF
+```
+
+### Listing AWS resources
+
+Wash also includes support for AWS. If you have your own and you've configured the AWS CLI on your workstation, you'll be able to use Wash to explore EC2 instances and S3 buckets.
+
+As an example, you might want to periodically check how many EC2 instances are running (and display that via [BitBar](https://getbitbar.com/)):
+```
+running=`find aws -action exec -meta .State.Name running 2>/dev/null | wc -l | xargs`
```

Comment:
I think this and the S3 bucket example would be more expressive via an `spath` primary. `spath` matches on the schema path, which are the labels joined by a `/`. So the EC2 example would be
```
find aws -spath '*ec2*instance' -meta .State.Name
```

and the S3 bucket example would be
```
find aws -spath '*s3*bucket'
```

Could file an issue if you think that'd be useful.

Reply:
I'm not against the spath primary, but I don't think it's a big improvement either.